onnxruntime/core/providers/cuda/tensor/resize_impl.h

```diff
@@ -23,7 +23,7 @@ void ResizeImpl(
     TArray<int64_t>& input_strides,
     TArray<fast_divmod>& output_div_pitches,
     TArray<float>& scales_vals,
-    TArray<float>& roi,
+    TArray<float, 10>& roi,
```

Comment:
default capacity is 8 which doesn't compile when input is 5D and roi vector contains 10 elements